tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>It feels like there's a paragraph missing here saying what weight clustering is and why you would want to use it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>@MarkDaoust: the idea for here and the other MOT tutorials is that they'll arrive at this page when they've already determined that they want to use weight clustering. If they reach this page without that, "If you want to see the benefits of weight clustering and what's supported, check the&nbsp;<a href="https://www.tensorflow.org/model_optimization/guide/clustering" target="_blank">overview</a>", etc. guides them to learn more. </p>

Reply:
Right LGTM.

I forgot to remove this comment after finding the other docs.